torch/_jit_internal.py

```diff
@@ -964,14 +964,14 @@ def container_checker(obj, target_type):
 
 
 def _isinstance(obj, target_type) -> bool:
-    origin_type = get_origin(target_type)    
```

Comment:
Maybe make these cosmetic changes in a separate PR?

Reply:
No idea why this change happened. I will remove them for now.